pubsub_nats_options.go

```diff
@@ -70,8 +68,14 @@ func natsOptClient(client natsClient) NATSOption {
 var ackMessage = []byte("ack")
 
 type natsSubscribeConfig struct {
-	queueGroup string
-	replier    func(msg *nats.Msg) []byte
+	queueGroup   string
+	replyTimeout time.Duration
+}
+
+func defaultSubscribeConfig() natsSubscribeConfig {
+	return natsSubscribeConfig{
+		replyTimeout: 60 * time.Second,
+	}
 }
```

Comment:
👀 @primalmotion 🔝 something like this yeah?

Reply:
yeah